packages/@dcl/inspector/src/lib/babylon/decentraland/gizmo-manager.ts

```diff
@@ -65,6 +71,11 @@ export function createGizmoManager(context: SceneContext) {
   gizmoManager.gizmos.positionGizmo!.updateGizmoRotationToMatchAttachedMesh = false
   gizmoManager.gizmos.rotationGizmo!.updateGizmoRotationToMatchAttachedMesh = true
 
+  // Configure all gizmos to only work with left click
+  if (gizmoManager.gizmos.positionGizmo) configureGizmoButtons(gizmoManager.gizmos.positionGizmo, [0])
```

Comment:
same as the previous comment, could we define this number as `const LEFT_BUTTON = 0`?